README.md

```diff
@@ -53,13 +53,9 @@ This documentation provides a comprehensive overview of the UKEF Digital TradeFi
 6. Set UKEF TFM environment variables in your terminal: `UKEF_TFM_API_SYSTEM_KEY` and `UKEF_TFM_API_REPORTS_KEY`.
 7. Run `npm run env:copy` to copy your root .env file into all the individual projects that need it.
 8. Run `npm ci` in the root folder of the repository. (note: this will install dependencies for the entire project, including those specified in sub-packages. More details on this in the [npm workspaces](./doc/npm-workspaces.md) docs)
-9. Start your local environment with `docker-compose up --build`.
-10. Create mock data
-    - If your work does not involve utilisation reports:
-      1. Create mock data by running `npm run load` from the root folder of the repository. This should generate mocks in your database (both Mongo and MSSQL). (for more details on what this does please see [utils docs](./utils/README.md)).
-    - If your work involves utilisation reports (including utilisation report E2E tests):
-      1. Run the command `npm run load:utilisation` from the root folder of the repository.
-         - This runs `npm run db:migrate -w libs/common` (see [SQL DB docs](./doc/sql-db.md#--run-migrations) for more details) in the `libs/common` directory which creates the SQL database tables. This is followed by `npm run load:sql` in the `utils` directory which inserts mock data for MongoDB and MSSQL. This is especially useful for utilisation report work and E2E tests
+9. Start your local environment with `npm run start`.
```

Comment:
@vamshidharukef Seems to have been a merge issue, can we please revert the non-E2E changes, this PR should only have changes in the E2E  directory